definitions/npm/axios_v0.19.x/flow_v0.80.x-/test_axios_v0.19.x.js

```diff
@@ -148,6 +148,11 @@ describe('Request methods', () => {
       .then(handleResponse)
       .catch(handleError);
 
+    axios
+      .options('anyUrl')
+      .then(handleResponse)
+      .catch(handleError);
```

Comment:
I think this will fail the review simply because we don't have the `it` and `describe` blocks for this new functionality. The old stuff would get grandfathered in, but any new tests would need to be under the new regime.

Reply:
If that's considered that old, this libdef is done back to September this year :), but I can certainly change it to describe and it style.

Reply:
Wait, if you expand the code block, you'll see it's actually wrapped in a describe it block already, so I guess we are good on that aspect.

Reply:
I hadn't seen that in greater context. So my new proposal is to break this into its own `it` block, because throwing so much stuff into a single `it` doesn't capture intent well.

I suggest using a `describe` for the `options` function itself, and `it` for individual aspects of `options` you wish to test. Furthermore, I would add a negative case for `options` since I could replace the definition with `any` and it would still pass this test.

Reply:
Fair enough, it's wrapped in a giant request methods block that's not easy to catch at a glance. I'll separate 'options' from it and make the addition more readable.